addons/web/static/src/views/kanban/kanban_controller.scss

```diff
@@ -1,54 +1,53 @@
 // ------- Kanban View Layout -------
 .o_kanban_renderer {
-    display: flex;
-    align-content: stretch;
+    // ----------------------------------------------------------------------------
+    // Customize the kanban view and its elements
+
+    // @param {number} --Kanban__record-width - Record width.
+    // @param {number} --Kanban__record-padding-v - Record vertical padding.
+    // @param {number} --Kanban__record-padding-h - Record horizontal padding.
+    // @param {number} --Kanban__record-margin-v - Record vertical margin.
+    // @param {number} --Kanban__record-margin-h - Record horizontal margin.
+    // @param {number} --Kanban__image-width - Regular image width.
+    // @param {number} --Kanban__imageFill-width - Filled image width.
+    // @param {number} --Kanban__innerElements-margin - Spacing between record inner elements.
+    // @param {number} --KanbanGroup__smallColumn-width - Small column width.
+    // @param {number} --KanbanGroup__padding-h - Column horizontal padding.
+    // @param {number} --KanbanGroup__paddingBottom - Column bottom padding.
+    // @param {number} --KanbanDashboard__record-padding-v - Dashboard record vertical padding.
```

Comment:
I think we should use the  `KanbanRecord-padding-v` variable directly `kanban_dashboard.scss`,  what do you think?
eg:

```scss

/// kanban_cotroller.scss
.o_kanban_renderer {
    --Kanban__record-padding-v: x;
}

/// kanban_dashboard.scss
.o_kanban_dashboard {
    --Kanban__record-padding-v: Y;
}